Gemfile

```diff
@@ -185,6 +185,6 @@ gem "avo-money_field"
 gem "avo-record_link_field"
 
 # gem "pagy", "< 8.0.0"
-gem "pagy", "> 8"
+gem "pagy", "> 9"
```

Comment:
We don't have to commit the Gemfile changes right now, to ensure that we don't break other apps.

Reply:
It's ok, this Gemfiles does not reflect on avo's dependencies, those are specified in `avo.gemspec`